lib/icons-to-woff.js

```diff
@@ -50,11 +53,9 @@ module.exports = function createIconFont (fs, icons, options) {
         }
         result = result.toString();
         const iconHeight = getSvgHeight(result, filename);
-        if (fontIconHeight && fontIconHeight !== iconHeight) {
-          return reject(new Error(`SVG font generation failed as not all icons have the same height. ` +
-            `Found: "${fontIconHeight}" and "${iconHeight}".`));
+        if (!iconHeight) {
+          return reject(new Error('SVG font generation failed as not all icons have a height.'));
```

Comment:
Please add the `filename` so the user knows which file has no height information